packages/pds/src/config/env.ts

```diff
@@ -9,6 +9,7 @@ export const readEnv = (): ServerEnvironment => {
     version: envStr('PDS_VERSION'),
     privacyPolicyUrl: envStr('PDS_PRIVACY_POLICY_URL'),
     termsOfServiceUrl: envStr('PDS_TERMS_OF_SERVICE_URL'),
+    acceptingImports: envBool('PDS_ACCEPTING_IMPROTS'),
```

Comment:
```suggestion
    acceptingImports: envBool('PDS_ACCEPTING_IMPORTS'),
```

Reply:
If we think there could be different kinds of imports in the future, could preemptively make it something like `PDS_ACCEPTING_REPO_IMPORTS`.

Reply:
yup i'm down 👍